leshan-server-redis/src/main/java/org/eclipse/leshan/server/redis/serialization/EndpointContextSerDes.java

```diff
@@ -74,27 +70,32 @@ public static JsonObject serialize(EndpointContext context) {
             }
         }
         /** copy the attributes **/
-        Map<String, Object> attributes = context.entries();
+        Map<Definition<?>, Object> attributes = context.entries();
```

Comment:
Is  there still a `EndpointContext` or `Attribute` serializer/deserialize ? 
I remember your first PR where you are using californium serializer/deserializer. 

This implementations seems backward compatible, correct ?

Reply:
There is still a binary serializer in Californium. Though json is used anyway, I implemented it in a more "backwards similar way".

Reply:
> This implementations seems backward compatible, correct ?

That's the intention, but not that deep tested :-).

Reply:
ok great :)